src/sass/bootstrap-asu-extends.scss

```diff
@@ -1,66 +1,44 @@
 // css Bootstrap doesn't have variables for
-@import '../components/accordions/accordions';// imports cards.
-@import '../components/alerts/alerts';
-@import '../components/block/block';
-@import 'extends/alt_styling';
-@import 'extends/anchor-menu';
-@import '../components/back-to-top/back-to-top';
-@import '../components/backgrounds/backgrounds';
+@import 'extends/misc';
```

Comment:
Looking at the bootstrap-asu-extends.scss, it appears we need to ensure updates to this file get made, but keep some custom additions, such as the imports of things in the `../components/` folder.